content/about/leaders-in-action.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I would rather not have SVG code in a Markdown file. I don't mind short oneliner SVGs in HTML, like a heart or a circle, but if it gets much longer than that, I prefer to have it in its own file.

There are a couple of ways to handle SVGs. If the SVG can be external, you can go on Almanack, click on the file manager and upload it to files.data.spotlightpa.org then link to it. 

If you need it to be on the page, you can save it in assets/img/logos/ and then add it to the list of on page SVGs [here](https://github.com/spotlightpa/poor-richard/blob/4ce422f0f5d00df3371dfe80dddee051f2647a16/layouts/_default/landing.html#L177-L189) by giving it an ID in the footer like this. 

```html
{{ $svgs := dict
    "leaders-in-action" "/img/logos/leaders-in-action.svg"
    "banner-svg" "/img/logos/banner-default-on-trans.svg"
    "bars-svg" "@fontawesome/solid/bars.svg"
    "calendar-svg" "@fontawesome/regular/calendar-check.svg"
    "close-svg" "@fontawesome/solid/circle-xmark.svg"
    "email-svg" "@fontawesome/regular/envelope.svg"
    "fb-svg" "@fontawesome/brands/facebook-f.svg"
    "logo-svg" "/img/logos/circle-default-on-trans.svg"
    "search-svg" "@fontawesome/solid/magnifying-glass.svg"
    "share-svg" "@fontawesome/solid/share.svg"
    "spinner-svg" "@fontawesome/solid/spinner.svg"
    "times-svg" "@fontawesome/solid/xmark.svg"
    "twitter-svg" "@fontawesome/brands/twitter.svg"
  }}
```

Then you can use it here like

```html
<svg class="float-right w-40 md:w-60 -mt-6 md:-mr-16 mb-2 ml-1 md:ml-2">
  <use href="#leaders-in-action" />
</svg>
```

Either way, I like to run SVGs through https://jakearchibald.github.io/svgomg/ to try to get them minified, although this one seems pretty small already.

I think since this is basically just an image, you should be able to get away with hosting it on files.data.spotlightpa.org, but I could be wrong.